### PR TITLE
x11-misc/gmrun: take over maint and bump version to 1.4w

### PR DESCRIPTION
**Pull request by henning-schild**

### CODE REVIEW

File: x11-misc/gmrun/gmrun-1.4w.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+fi
+
+DESCRIPTION="A GTK-2 based launcher box with bash style auto completion!"
+HOMEPAGE="https://github.com/WdesktopX/gmrun/"
+
+LICENSE="GPL-1"
+SLOT="0"
+IUSE="elibc_glibc"
+
+RDEPEND="x11-libs/gtk+:2
```
Comment (henning-schild) [inline x11-misc/gmrun/gmrun-1.4w.ebuild:22]: the new one supports gtk 2 and 3, that still needs to be dealt with

File: x11-misc/gmrun/gmrun-1.4w.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	EGIT_BRANCH="master"
+else
+	SRC_URI="https://github.com/WdesktopX/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~mips ~ppc x86"
+fi
+
+DESCRIPTION="A GTK-2 based launcher box with bash style auto completion!"
+HOMEPAGE="https://github.com/WdesktopX/gmrun/"
+
+LICENSE="GPL-1"
```
Comment (henning-schild) [inline x11-misc/gmrun/gmrun-1.4w.ebuild:18]: this seems incorrect on that new fork, seems like the author invented his/her own license

Comment (henning-schild) [reply x11-misc/gmrun/gmrun-1.4w.ebuild:18]: it is "ISC" 

https://github.com/WdesktopX/gmrun/commit/ed2760ac9745c7f7b3aecf850a590e683786a57a

File: x11-misc/gmrun/gmrun-1.4w.ebuild (hunk 1 above)
Comment (henning-schild) [reply x11-misc/gmrun/gmrun-1.4w.ebuild]: going gtk3 only

**Comment by bircoph** [comment]
This package could use some USE flags (see configure options):
- nls for native language support
- xdg for xdg-compatible paths (but this will break compatibility with history file location of the old version, don't enable it by default).

Also you should explicitly disable gtk2 and enable gtk3 via configure options.

And please add README.md to dodoc, not README (there is no such file).

**Comment by henning-schild** [comment]
> And please add README.md to dodoc, not README (there is no such file).

It was "README*" but well, why not make it "README.md"

**Comment by henning-schild** [comment]
> * nls for native language support
> 
>     * xdg for xdg-compatible paths (but this will break compatibility with history file location of the old version, don't enable it by default).
> 
> 
> Also you should explicitly disable gtk2 and enable gtk3 via configure options.

good points, done

**Comment by slashbeast** [comment]
looks good to me, will merge.

**Pull request closed without merging**